contracts/SnappAuction.sol

```diff
@@ -1,4 +1,5 @@
 pragma solidity ^0.5.0;
+// solium-disable security/no-block-members
```

Comment:
Should this be part of the `.soliumrc.json` instead of putting it into every contract?

Reply:
I would like it to be, but haven't figured out how yet!

Reply:
Does
```
"rules": {
  <property>: "off"
}
```

as shown [here](https://github.com/jbaylina/ERC820/blob/master/.soliumrc.json) not work?

Reply:
Worked! Thanks!